index.js

```diff
@@ -52,25 +52,17 @@ class LokiTransport extends Transport {
     })
 
     // Deconstruct the log
-    const { label, labels, timestamp, level, message, ...rest } = info
+    const { timestamp } = info
 
     // build custom labels if provided
-    let lokiLabels = { level: level }
+    let lokiLabels = {}
 
     if (this.labels) {
       lokiLabels = Object.assign(lokiLabels, this.labels)
-    } else {
-      lokiLabels.job = label
     }
 
-    lokiLabels = Object.assign(lokiLabels, labels)
-
     // follow the format provided
-    const line = this.useCustomFormat
-      ? info[MESSAGE]
-      : `${message} ${
-      rest && Object.keys(rest).length > 0 ? JSON.stringify(rest) : ''
-    }`
+    const line = this.useCustomFormat ? info[MESSAGE] : JSON.stringify(info)
```

Comment:
`useCustomFormat` is based on the fact whether some formats have been passed to the transport or not. But only [finalizing formats](https://github.com/winstonjs/logform) would add `MESSAGE` to the `info`, e.g. if only `timestamp` format would be used, then `MESSAGE` would be empty. How about completely removing `useCustomFormat` and replacing the check here with
```
const line = MESSAGE in info ? info[MESSAGE' : JSON.stringify(info)
```
?

Reply:
I'm not very familiar with Typescript or winston, please keep that in mind when reading the following comment ;).

From my understanding `MESSAGE` is mandatory and [never undefined](https://github.com/winstonjs/logform#info-objects), and contains what the user passes as log message, when calling `logger.log(level,message)`. 
And from what I understand, the initial intent with the `format` parameter for this transport is to allow to customize the log message with winston format and not have the transport change anything. As a user of the transport I'd expect it to respect this contract and see in Loki the same output as I would see in the console for example.
By default and as fallback we make it a JSON string because that's a format that's well supported by Loki by default with the `| json` parser.

If we're going with your suggestion, we'd loose the `level` and other metadata like labels, when not specifying any format and I think that'd be confusing. One would then have to use the [json format](https://github.com/winstonjs/logform#json) or another format to not loose the metadata, which basically makes the `format` option mandatory. What do you think?

Reply:
Yes, according to the docs, `MESSAGE` is "complete string message set by "finalizing formats": json, logstash, printf, prettyPrint, simple" but after some testing, it appeared that you were correct - these formats are not setting the value but overriding some default one.

I think the best way would be to discuss this in the [separate issue](https://github.com/JaniAnttonen/winston-loki/issues/116), outside of this PR.